src/components/views/styl/Home.styl

```diff
@@ -9,47 +11,45 @@
     justify-content: space-between;
     line-height: 1.5;
     margin: 0;
-}
-
-.explanations.alt dt {
-    border: 1px solid #efefef;
-    font-weight: 800;
-}
 
-.explanations.alt dt, .explanations.alt dd {
-    padding: 10px;
-}
-
-.explanations.alt dd {
-    background-color: #efefef;
+    &.alt {
+        dt {
+            border: 1px solid #efefef;
+            font-weight: 800;
+        }
+        dt, dd {
+            padding: 10px;
+        }
+        dd {
+            background-color: #efefef;
+        }
+    }
 }
 
 .highlighted {
     margin: 20px 0;
 }
 
-@media all and (min-width: 550px) {
+@media $base-tablet {
     .explanations {
         flex-direction: row;
-    }
-
-    .explanations.alt dd {
-        border-right: 1px solid #fff;
-    }
-
-    .explanations.alt dt {
-        border-left: 0;
-    }
-
-    .explanations.alt div:first-child dt {
-        border-left: 1px solid #efefef;
-    }
-
-    .explanations.alt div:last-child dd {
-        border-right: 1px solid #efefef;
-    }
 
-    .explanations.split > div {
-        width: 45%;
+        &.alt {
+            dd {
+                border-right: 1px solid #fff;
+            }
+            dt {
+                border-left: 0;
+            }
+            div:first-child dt {
+                border-left: 1px solid #efefef;
+            }
+            div:last-child dd {
+                border-right: 1px solid #efefef;
```

Comment:
Also here? To be honest I haven't double-checked that these values are all related, even though they are equal.